rules/crowdstrike_rules/crowdstrike_base64_encoded_args.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from panther_base_helpers import crowdstrike_process_alert_context
+
+
+def rule(event):
+    # List of command line tools to monitor for execution with Base64 encoded arguments
+    command_line_tools = {
+        "powershell.exe",
+        "cmd.exe",
+        "cscript.exe",
+        "wscript.exe",
+        "rundll32.exe",
+    }
+
+    # Define a regular expression pattern to match Base64 encoded strings
+    base64_pattern = re.compile(r"[A-Za-z0-9+/]{10,}[=]{0,2}")
+
```

Comment:
Should check if this is a windows event like in the other detections as well.

Reply:
Added if statement to check for platform. We can eventually add in support for Lin/Mac platform on this rule.